godtools/App/Services/Renderer/BackgroundImageRenderer/MobileContentBackgroundImageRendererType.swift

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2021 Cru. All rights reserved.
 //
 
-import Foundation
+import UIKit
```

Comment:
it looks like there was some code updates that got included with this PR. I'm guessing that wasn't intentional.

Are these updates something that should be handled in a different PR, or are they changes that don't matter being bundled here?

Reply:
nm, I see the commit. you were fixing the tests